featuretools/primitives/standard/rolling_transform_primitive.py

```diff
@@ -251,13 +386,20 @@ def __init__(self, window_length=3, gap=0, min_periods=1):
         self.gap = gap
         self.min_periods = min_periods
 
+    def _pandas_std(self, series):
+        return series.std()
```

Comment:
Tried using `np.std`, but it seems like there's a slightly different handling of nans that produces different results from the pandas implementation

Reply:
If you want, you can define this as a function inside the scope of `rolling_std` since that is the only place it's used, rather than adding it as a class method.

Reply:
that makes sense! changing